rpcs3/rpcs3qt/shortcut_settings.cpp

```diff
@@ -44,7 +43,6 @@ shortcut_settings::shortcut_settings()
 		{ shortcut::mw_toggle_fullscreen, shortcut_info{ "main_window_toggle_fullscreen", tr("Toggle Fullscreen"), "Alt+Return", shortcut_handler_id::main_window } },
 		{ shortcut::mw_exit_fullscreen, shortcut_info{ "main_window_exit_fullscreen", tr("Exit Fullscreen"), "Esc", shortcut_handler_id::main_window } },
 		{ shortcut::mw_refresh, shortcut_info{ "main_window_refresh", tr("Refresh"), "Ctrl+F5", shortcut_handler_id::main_window } },
-		{ shortcut::mw_welcome_dialog, shortcut_info{ "main_window_welcome_dialog", tr("Show Welcome Dialog"), "F1", shortcut_handler_id::main_window } },
```

Comment:
Why are you removing this?

Reply:
it conflicts with debugget help shortcut, I also don't think there is a need for a shortcut to the welcome dialog. It is a welcome dialog after all, should be displayed once and if the user wants to see it again it's in the menu.